visualization/app/material/_theme.scss

```diff
@@ -1,8 +1,17 @@
-@use "~@angular/material" as mat;
+@use "@angular/material" as mat;
 
-@import "~@angular/material/theming";
+@import "@angular/material/theming";
 
-@include mat.core();
+// TODO(v15): As of v15 mat.legacy-core no longer includes default typography styles.
```

Comment:
Should this be addressed in this PR? The v15 comment indicates for me that this might be the intention?

Reply:
it is an automatically added todo by the ng migration tool. I thought about addressing it, but it will maybe come with some further code changes and different look. Therefore I added a todo in #3116 for it to be addressed.